### PR TITLE
Fluent: Scheduler - Make a circle around today date (apply dxCalendar style)

### PR DESCRIPTION
**Pull request by markallenramirez**

### CODE REVIEW

File: packages/devextreme/js/renovation/ui/scheduler/workspaces/base/header_panel/date_header/dateHeaderText.tsx [hunk 1]
```diff
@@ -17,7 +17,7 @@ export const viewFunction = ({
     {
       splitText
         // eslint-disable-next-line react/jsx-key
-        ? textParts.map((part) => (<span className="dx-scheduler-header-panel-cell-date">{part}</span>))
+        ? textParts.map((part) => (<div className="dx-scheduler-header-panel-cell-date"><span>{part}</span></div>))
```
Comment (andrewmakarov) [inline packages/devextreme/js/renovation/ui/scheduler/workspaces/base/header_panel/date_header/dateHeaderText.tsx:20]: What a reason change tag to <div />?

File: packages/devextreme/js/renovation/ui/scheduler/workspaces/base/header_panel/date_header/layout.tsx [hunk 2]
```diff
@@ -73,7 +73,7 @@ export const viewFunction = ({
               dateCellTemplate={dateCellTemplate}
               key={key}
               colSpan={colSpan}
-              splitText={isMaterial}
+              splitText={isMaterial || isFluent}
```
Comment (andrewmakarov) [inline packages/devextreme/js/renovation/ui/scheduler/workspaces/base/header_panel/date_header/layout.tsx]: I have already seen several uses 'isMaterial || isFluent', maybe you can make a common uses line 'isMaterialBased'?

File: packages/devextreme/scss/widgets/fluent/scheduler/_index.scss [hunk 3]
```diff
@@ -220,9 +220,26 @@ $fluent-scheduler-agenda-time-panel-cell-padding: 8px;
 
 
   &.dx-scheduler-header-panel-current-time-cell {
+    color: $scheduler-panel-text-color;
+
     &::before {
       content: none;
     }
+
+    > div:last-child {
```
Comment (andrewmakarov) [inline packages/devextreme/scss/widgets/fluent/scheduler/_index.scss]: Does it really make sense to have such a strict selector?

File: packages/devextreme/js/renovation/ui/scheduler/workspaces/base/header_panel/date_header/dateHeaderText.tsx (hunk 1 above)
Comment (Raushen) [reply packages/devextreme/js/renovation/ui/scheduler/workspaces/base/header_panel/date_header/dateHeaderText.tsx:20]: I think a difference in a display style: was inline, became block.

Comment (markallenramirez) [reply packages/devextreme/js/renovation/ui/scheduler/workspaces/base/header_panel/date_header/dateHeaderText.tsx:20]: The following style changes the span element's display style to `block`: [_indes.scss:229](https://github.com/DevExpress/DevExtreme/blob/23_2/packages/devextreme/scss/widgets/fluent/scheduler/_index.scss#L229).

The current solution requires changing the element's display style to `inline-flex`: [_index.scss:233](https://github.com/DevExpress/DevExtreme/pull/25558/files#diff-7dd61f89332668e9ffe9f2c7d6f9d8c61ff124eceab037c8783da6277bda79faR233). The most noticeable effect by this change is that the `line-height` style (see [_index:243](https://github.com/DevExpress/DevExtreme/blob/23_2/packages/devextreme/scss/widgets/fluent/scheduler/_index.scss#L243)) is no longer applied causing inconsistencies in a header cell layout.

By wrapping a span element with a div element, it is easier to apply styles to the span element without affecting a header cell layout.

File: packages/devextreme/scss/widgets/fluent/scheduler/_index.scss (hunk 3 above)
Comment (markallenramirez) [reply packages/devextreme/scss/widgets/fluent/scheduler/_index.scss]: Not really. Refactored.

**Review by Raushen — DISMISSED** [review]

**Review by Raushen — APPROVED** [review]

**Review by IlyaKhD — APPROVED** [review]

**Pull request merged**